modules/users/client/components/UsersList.js

```diff
@@ -0,0 +1,33 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import styled from 'styled-components';
+
+// Internal dependencies
+import UserSummary from './UserSummary';
+
+export default function UsersList({ users }) {
+  const UsersList = styled.div`
```

Comment:
Now, this is nitpick: We could consider using `ul` and `li` to identify a list in html.